docs/flows.md

```diff
@@ -175,13 +193,20 @@ Sometimes when using 3rd party SDKs that that are not instrumented for observabi
 ```js
 import { defineFlow, run } from '@genkit-ai/flow';
 
-export const myFlow = defineFlow(
-  { name: 'myFlow', inputSchema: z.string(), outputSchema: z.string() },
+export const menuSuggestionFlow = defineFlow(
+  {
+    name: 'menuSuggestionFlow',
+    inputSchema: z.object({ cuisine: z.string() }),
+    outputSchema: z.string(),
+  },
   async (input) => {
-    const output = await run('step-name', async () => {
-      return await doSomething(input);
+    const cuisines = await run('find-similar-cusines', async () => {
```

Comment:
spell check; should be `find-similar-cuisines`

Reply:
Done.